The-Chonky-Panda/Atelier
# Clicking on styles changes image gallery, price, and sizes

## Description
Clicking on any of the style thumbnail images will update the currentStyle property in our store, re-rendering the image gallery as well as the price, style name, and size options.

## How to test
Manually, open up the app, click on the style thumbnails and make sure that the images in the image gallery update.

## Notes
- Also added style to the size and qty dropdowns and add to cart button
- Some product's styles do not have differing thumbnail images compared to the product's other styles. Manual testing won't seem like its working on these products (product_id 47421 works fine)
- `ImageGallery.jsx` code is pretty sloppy so definitely planning on refactoring

## Issue tracking
https://trello.com/c/f4RYbJaf/49-s-clickable-style-option
